src/codeu/chat/common/UserType.java

```diff
@@ -42,31 +42,25 @@ private static UserType fromId(int id) {
     }
     return ut;
   }
-  
+
   public static UserType fromString(String ut) {
-	switch(ut) {
-	  case "CREATOR":
-	    return UserType.CREATOR;
-	  case "OWNER":
-		return UserType.OWNER;
-	  case "MEMBER":
-		return UserType.MEMBER;
-	  case "NOTSET":
-		return UserType.NOTSET;
-	  default:
+    switch (ut) {
+      case "CREATOR":
+        return UserType.CREATOR;
+      case "OWNER":
+        return UserType.OWNER;
+      case "MEMBER":
+        return UserType.MEMBER;
+      case "NOTSET":
+        return UserType.NOTSET;
+      default:
         System.out.println("ERROR: invalid string");
         return null;
-	}
+    }
   }
-  
+
   // returns true if the user has manager access for the target UserType
   public static boolean hasManagerAccess(UserType user, UserType target) {
-	if (user == UserType.CREATOR || target == UserType.NOTSET) {
-	  return true;
-	}
-	if (user == UserType.OWNER && target == UserType.MEMBER) {
-	  return true;
-	}
-	return false;
+    return user.fId < target.fId;
```

Comment:
For now this is okay, but I would prefer that the user type hierarchy be separate from the internal representation used for serialization.